### ISSUE TITLE
harmless syslog errors when running tests on MacOS

### ISSUE THREAD

**Issue by zhonglushu** [vitessio/vitess#3165]
When I run zk-up.sh, it tells me that can not connect to syslog, words like this:

```
Starting zk servers...
Waiting for zk servers to be ready...
Started zk servers.
ERROR: logging before flag.Parse: E0912 14:33:17.160626     142 syslogger.go:122] can't connect to syslog
ERROR: logging before flag.Parse: E0912 14:33:17.242043     152 syslogger.go:122] can't connect to syslog
ERROR: logging before flag.Parse: E0912 14:33:17.349889     162 syslogger.go:122] can't connect to syslog
W0912 14:33:17.354957     162 vtctl.go:91] cannot connect to syslog: Unix syslog delivery error
Configured zk servers.
```

My docker is lastest [docker for mac](https://www.docker.com/docker-mac), vitess images is [vitess/base](https://hub.docker.com/r/vitess/base/).

**Comment by m-masataka** [comment]
Did you run vitess in container? 
I think the course of these errors is that syslogd does't start in your container.
So you have to start up syslog daemon to avoid these errors.
But these errors are negligible term, if you just want to try vitess.

**Comment by michael-berlin** [comment]
I saw these errors as well when I ran tests on my Mac. Not sure if I ran them natively or within Docker.

In any case, like @m-masataka said these errors can be ignored.

It's unlikely that we'll spend time fixing this. Closing this issue for now. I'll also rename the issue to clarify the impact.

**Comment by sougou** [comment]
Reopening per this thread: https://groups.google.com/forum/#!topic/vitess/CQDL3aY-q_E. We'll keep it open till the bug is fixed.

**Comment by gedgar** [comment]
This looks like it's fixed by [#3563](https://github.com/vitessio/vitess/issues/3563).
